frontend/src/components/pages/CreateAccountPage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Text, Button } from "@chakra-ui/react";
+import { ReactComponent as Logo } from "../../assets/Sistering_Logo.svg";
+
+const CreateAccountPage = (): React.ReactElement => {
+  const onContinue = () => {
+    //
+  };
+
+  return (
+    <div
+      style={{
+        width: "100%",
+        display: "flex",
+        flexDirection: "row",
+        height: "100vh",
+      }}
+    >
```

Comment:
Please use the Chakra UI `Box` component instead of vanilla `div` we want to stay consistent in our style.
More info here: https://chakra-ui.com/docs/components/layout/box

```suggestion
    <Box
        width="100%"
        display="flex"
        flexDirection="row"
        height="100vh"
      }}
    >
```

same with the rest of the `div` usage in the file.